flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -635,8 +644,18 @@
 	 * The default number of archived jobs for the jobmanager
 	 */
 	public static final int DEFAULT_JOB_MANAGER_WEB_ARCHIVE_COUNT = 5;
-	
-	
+
+	/**
+	 * By default, submitting jobs from the web-frontend is allowed.
+	 */
+	public static final boolean DEFAULT_JOB_MANAGER_WEB_SUBMISSION = true;
+
+	/**
+	 * Default directory for uploaded file storage for the Web frontend.
+	 */
+	public static final String DEFAULT_JOB_MANAGER_WEB_UPLOAD_DIR =
+			(System.getProperty("java.io.tmpdir") == null ? "/tmp" : System.getProperty("java.io.tmpdir")) + "/webmonitor/";
```

Comment:
I thought this was considered a feature of the webclient that jars were persisted. I personally do not like it either. Should I do this then?